engine/flink/management/periodic/src/test/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessServiceTest.scala

```diff
@@ -68,6 +75,13 @@ class PeriodicProcessServiceTest
     val events                        = new ArrayBuffer[PeriodicProcessEvent]()
     val additionalData                = Map("testMap" -> "testValue")
 
+    implicit val deploymentService: ProcessingTypeDeploymentService with ActionIdChecker =
```

Comment:
KISS. Why not just use concrete type (`ProcessingTypeDeploymentServiceStub`) instead?